src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -22,7 +22,6 @@
 import static com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck.MSG_JAVADOC_MISSED_HTML_CLOSE;
 import static com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck.MSG_JAVADOC_PARSE_RULE_ERROR;
 import static com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck.MSG_JAVADOC_WRONG_SINGLETON_TAG;
-import static com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck.MSG_KEY_UNRECOGNIZED_ANTLR_ERROR;
```

Comment:
Yes it shouldn't be possible any longer. This one at [messages.properties#L1](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties#L1).

The reason I didn't remove `unrecognized` one is that right now it looks like that `report` should be called under all circumstances. But it might so happen that under some weird input or for some other reason ANTLR fails to report to listener and then we might again require it. So I thought that after some time of usage when we can be sure that never does ANTLR fail to report to listener, we can remove all the unutilized message keys together in one issue so that all other message.properties get cleaned.